src/test/java/com/kodilla/ecommercee/entity/CartRepositoryTest.java

```diff
@@ -50,14 +51,17 @@ public void testSaveAndGetCart() {
         // When
         System.out.println("Test get all records ...\n");
         cartRepository.save(cartA);
+        Long firstId = cartA.getId();
         cartRepository.save(cartB);
         cartRepository.save(cartC);
 
-        List<Cart> carts = cartRepository.findAll();
+        Cart cart = cartRepository.getOne(firstId);
+        ArrayList<Cart> newList = new ArrayList<>();
+        newList.add(cart);
 
         // Then
-        assertTrue(carts.size() >= 3);
-        assertTrue(carts.contains(cartB));
+        assertTrue(cartRepository.count() == 3);
```

Comment:
Używajmy do takiego porównania assertEquals zamiast assertTrue. Będzie to czytelniejsze w przypadku błędu